src/script/common/c_content.cpp

```diff
@@ -526,6 +526,12 @@ ContentFeatures read_content_features(lua_State *L, int index)
 	// Amount of light the node emits
 	f.light_source = getintfield_default(L, index,
 			"light_source", f.light_source);
+	if (f.light_source > LIGHT_MAX) {
+		warningstream << "Node " << f.name.c_str()
+			<< " had greater light_source than 14, it was reduced."
```

Comment:
Could you change this to "had light_source > " << LIGHT_MAX << ", it was reduced" instead?

LIGHT_MAX very well could change if/when I (or somebody else!) ever get around to implementing the RGB lighting model.